pyproject.toml

```diff
@@ -1,15 +1,13 @@
 [build-system]
-requires = ["setuptools", "wheel"]
-# uncomment to enable pep517 after versioneer problem is fixed.
-# https://github.com/python-versioneer/python-versioneer/issues/193
-# build-backend = "setuptools.build_meta"
+requires = ["setuptools", "wheel", "versioneer"]
+build-backend = "setuptools.build_meta"
```

Comment:
At the end of the day, this is what fixed it. The `build-system` provides a standard way of building the tarball and wheel ([PEP 517](https://peps.python.org/pep-0517/)).